src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -7569,8 +7569,7 @@ private static Encoding GetEncodingFromEnum(FileSystemCmdletProviderEncoding typ
 
                 case FileSystemCmdletProviderEncoding.Oem:
                     {
-                        uint oemCP = NativeMethods.GetOEMCP();
-                        encoding = System.Text.Encoding.GetEncoding((int)oemCP);
+                        encoding = ClrFacade.GetOEMEncoding();
```

Comment:
`ClrFacade` has its own copy of `GetOEMEncoding`, so the one in `FileSystemProvider.cs` can be removed.